src/libaudqt/audqt.cc

```diff
@@ -35,6 +41,33 @@ namespace audqt {
 
 static int init_count;
 
+// wrap QApplication only so we can generate a debug trace
+class AudApplication : public QApplication
+{
+public:
+    AudApplication (int &argc, char **argv)
+        : QApplication (argc, argv)
+    {
+        setAttribute (Qt::AA_UseHighDpiPixmaps);
+#if QT_VERSION >= QT_VERSION_CHECK(5, 3, 0)
+        setAttribute (Qt::AA_ForceRasterWidgets);
+#endif
+
+        setApplicationName (_("Audacious"));
+        if (windowIcon().isNull()) {
+            QIcon appIcon = get_icon (argv[0]);
+            setWindowIcon (appIcon);
+        }
+        setQuitOnLastWindowClosed (true);
```

Comment:
What is the setQuitOnLastWindowClosed() for?  There are some cases where we want to keep running without any open windows, for example the minimize-to-status-icon feature.

Reply:
Not merging this without a good explanation of why it's necessary.  We handle quitting already through aud_quit() when windows are closed.